python-frontend/typeshed_serializer/serializer/serializers.py

```diff
@@ -105,8 +121,12 @@ def serialize(self, output_dir_name="output") -> None:
                 continue
             current_file = build_result.files.get(file)
             module_symbol = symbols.ModuleSymbol(current_file)
-            symbols.save_module(module_symbol, self.output_folder, is_debug=self.is_debug,
-                                debug_dir=output_dir_name)
+            symbols.save_module(
```

Comment:
Not sure if the formatting changes are really necessary, but it does look better